site/content/docs/5.0/migration.md

```diff
@@ -7,6 +7,12 @@ aliases: "/migration/"
 toc: true
 ---
 
+## v5.0.0
+
+### JavaScript
+
+- Dropdowns now have one more option (`autoClose`) to handle the auto close behavior. You can use this option to accept the click inside or outside the dropdown menu to make it interactive.
```

Comment:
Should we maybe link to an example? Do we have one?

Reply:
Yes here: https://deploy-preview-33770--twbs-bootstrap.netlify.app/docs/5.0/components/dropdowns/#auto-close-behavior

Reply:
Can you link to it somehow? We already do this for some other entries and I believe it helps.